examples/run_script_with_local_browser/README.md

```diff
@@ -0,0 +1,37 @@
+# Example script: run the script in existing local browser with AgentQL
+
+This example demonstrates how to run the script in an existing local browser with AgentQL.
```

Comment:
Not necessary local, can be remote. Main thing is that it's not bundled.
Please, fix this in other places too.

```suggestion
This example demonstrates how to interact with an external or existing browser with AgentQL.
```